Gopkg.toml

```diff
@@ -0,0 +1,3 @@
+[prune]
+  go-tests = true
+  unused-packages = true
```

Comment:
You may not want this. It makes development more difficult by forcing you to run dep ensure every time you use a new package rather than once.